framework/meta/src/cmd/standalone/upgrade/upgrade_print.rs

```diff
@@ -10,7 +13,9 @@ pub fn print_upgrading(dir: &RelevantDirectory) {
         println!(
             "\n{}",
             format!(
-                "Upgrading from {from_version} to {to_version} in {}\n",
+                "Upgrading from {} to {} in {}\n",
+                from_version.version,
```

Comment:
We can implement the `Display` trait implemented for FrameworkVersion, and then have from_version/to_version printed directly.